xlators/mgmt/glusterd/src/glusterd-brick-ops.c

```diff
@@ -2130,17 +2107,10 @@ glusterd_op_add_brick(dict_t *dict, char **op_errstr)
 {
     int ret = 0;
     char *volname = NULL;
-    glusterd_conf_t *priv = NULL;
     glusterd_volinfo_t *volinfo = NULL;
-    xlator_t *this = NULL;
     char *bricks = NULL;
     int32_t count = 0;
-
-    this = THIS;
-    GF_ASSERT(this);
-
-    priv = this->private;
-    GF_ASSERT(priv);
```

Comment:
We should not remove GF_ASSERT check for priv, in case of THIS is NULL glusterfs_this_location sets global xlator to THIS and global_xlator does not have valid private object.